tests/plotting/test_colors.py

```diff
@@ -121,3 +122,187 @@ def test_color():
 def test_color_opacity():
     color = pv.Color(opacity=0.5)
     assert color.opacity == 128
+
+
+# Define standardized names and hex values of web colors defined here:
+# https://www.w3.org/TR/css-color-4/#named-colors
+WEB_COLORS: final = {
+    'aliceblue': '#F0F8FF',
```

Comment:
It's hard to check if these color codes are correct, so it's a good idea to use the Matplotlib function. However, we need to check that Matplotlib covers all of the codes here.
```python
>>> from matplotlib import colors
>>> print(colors.cnames['aliceblue'])
#F0F8FF
```

Reply:
Fixed in https://github.com/pyvista/pyvista/pull/6593/commits/fe593cf7249433a96b275ba5fc3be392fbb6f98e